Script to install/uninstall packages

AbdullahM0hamed opened the pull request:
This script calls the package installer via Intents and requires the following permissions to be in termux-api:

`android.permission.REQUEST_INSTALL_PACKAGES`
`android.permission.REQUEST_DELETE_PACKAGES`

AbdullahM0hamed commented:
@xalexalex, it is a typo, yes.

I couldn't figure out how to change it though, unfortunately.

AbdullahM0hamed commented:
I'm not sure what I was doing wrong before, but I've managed to change the name. I've also rearranged the logic as requested.

nikitabobko commented on scripts/termux-apk:
```diff
@@ -0,0 +1,50 @@
+
+install()
+{
+    apk="$(realpath $@)"
```
This function is supposed to be called with only one argument. But, `$@` gives false sense that this function can be called with several arguments. I'd better change it to `$1`. Or, which is certainly preferable I'd add support for several args passed to `-i` and `-u` and in this function iterate over `$@`

nikitabobko commented on scripts/termux-apk:
```diff
@@ -0,0 +1,50 @@
+
+uninstall()
+{
+    am start -a android.intent.action.UNINSTALL_PACKAGE -d package:"$@"
```
The same about `$@` here

nikitabobko commented on scripts/termux-apk:
```diff
@@ -0,0 +1,50 @@
+	echo "$apk does not exist"
+	exit 1
+    fi
+    am start -a android.intent.action.INSTALL_PACKAGE -d "file://$apk"
```
I'd propagate exit code of `am` command with `return $?` and put `set -e` at the start of the script. The same about `uninstall` function

nikitabobko commented on scripts/termux-apk:
```diff
@@ -0,0 +1,50 @@
+
+if [ $# != 2 ]
+then
+       show_usage
```
What's wrong with codestyle? it's 8 spaces. But at the same time you use 4 spaces in other places in this file. Also tabs mixed with spaces across the file

AbdullahM0hamed commented:
@nikitabobko, I've addressed your concerns. Regarding that last concern I edited this using emacs (in termux) and I haven't changed anything from the default settings (I went ahead an manually backspaced it through).

Movado04 commented:
How can I use my termux
